index.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line
+
+import displayBooks from './modules/bookManager';
+
+import './modules/display';
+
+import './modules/time';
```

Comment:
- [ ] 🤔🔍 As the previous reviewer mentioned, your app doesn't work as expected. When I click to navigate, nothing happens. 🚫🧭 Kindly investigate this issue, as it prevented me from testing your app's functionalities. 🕵️‍♂️ I believe the reason behind this problem might be that your imported file lacks the extension `.js`. 📃💡 To adhere to JavaScript best practices, please update your code and include the proper file extensions in your imports. 🔄🚀 By doing so, you can resolve this issue and ensure smooth navigation within your app. 🛠️😊 Keep up the good work, and I'm looking forward to testing your app's functionalities! 🌟📲